### DISCUSSION TITLE
Added Direct Award Justification codes, can they be added to the documentation.

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#1084] [Q&A] [answered]
There the SDK (i am not sure about the version) there were 3 Direct Award Justification codes added: exc-circ-rail, char-imp, sim-infra.
But they were not added to the documentation. Especially in the table Table 12. Direct Award Justification code context use

Can you add these codes to the documentation (in table 12 where there is shown which direct award justification codes are relevant for which directive, see https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html#acceleratedProcedureSection))? So that we can implement these 3 codes. 

![image](https://github.com/user-attachments/assets/f68aa651-b6fc-4b5f-a93e-2bf528bb78ba)

**Comment by YvesJo** [comment, marked as answer]
Hi,
Some added codes are specific to T01 and T02 and were not added to the documentation, they however are part of the list of codes.
There is also an ongoing activity to clarify the restrictions for the use of the various codes.
Documentation will get updated once this clarified.
KR

**Comment by JvdWalNL** [comment]
@YvesJo can you please give an update on the clarification proces? Hence the fix is not mentioned in SDK 1.14 this last friday..

**Reply by YvesJo** [reply]
Hi,
This requires the greatest care as this would directly affect the available values for each form. It has not been possible to spend the required time to address this with SDK 1.14.
KR

**Reply by JvdWalNL** [reply]
Hi,

Can you give some expectation in which SDK it will be fixed? This issue is open since dec 2024.. Therefore the SDK is still not in line with regulations..

Kind Regards
